R/spark_compile.R

```diff
@@ -126,7 +126,7 @@ spark_compile <- function(name,
 #'
 #' @keywords internal
 #' @export
-spark_compile_package_jars <- function(package = rprojroot::find_package_root_file(),
+compile_package_jars <- function(package = rprojroot::find_package_root_file(),
```

Comment:
Okay, let's leave it as jars then!

On Fri, Aug 5, 2016 at 10:01 PM, Kevin Ushey <notifications@github.com>
wrote:

> In R/spark_compile.R
> <https://github.com/rstudio/sparklyr/pull/150#discussion_r73778847>:
>
> > @@ -126,7 +126,7 @@ spark_compile <- function(name,
> >  #'
> >  #' @keywords internal
> >  #' @export
> > -spark_compile_package_jars <- function(package = rprojroot::find_package_root_file(),
> > +compile_package_jars <- function(package = rprojroot::find_package_root_file(),
>
> I could go either way on the function name as well (technically, we'll be
> generating one jar for each combination of Spark version * scala version
> provided)
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/rstudio/sparklyr/pull/150/files/dafaaa2516e9a03fac9075f1222cb9285069e69a..26772d833cb691b37a37157f0c62a54a961be082#r73778847>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAGXx3ZidQ8Ir85xEBA47k9m2Fml3jN2ks5qc-sMgaJpZM4JeE4J>
> .
>